templates/terraform/processing-project/service-accounts.tf

```diff
@@ -12,9 +12,17 @@ module dataflow_runner_account {
 }
 
 resource google_project_iam_member command_center_argo_account_iam {
+  provider = google.target
   for_each = toset(["dataflow.developer", "compute.viewer", "bigquery.jobUser"])
 
-  provider = google.target
   role = "roles/${each.value}"
   member = "serviceAccount:${var.command_center_argo_account_email}"
 }
+
+resource google_service_account_iam_binding dataflow_runner_user_binding {
+  provider = google.target
+
+  service_account_id = module.dataflow_runner_account.id
+  role = "roles/iam.serviceAccountUser"
+  members = ["serviceAccount:${var.command_center_argo_account_email}"]
+}
```

Comment:
Undocumented requirement needed for the argo-runner account to act as the dataflow-runner account.

Reply:
Want to add a comment explaining why it is needed since it is undocumented?

Reply:
Documentation wot